docusaurus/docs/guides/Identity_Providers/index.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Identity Providers for BrowZer
+sidebar_label: Identity Providers for BrowZer
```

Comment:
This `sidebar_label` is currently being ignored because docusaurus/docs/guides/Identity_Providers/_category_.yml has `link.id: guides/Identity_Providers/index`, so that category's `label` property has precedence.